vendor/github.com/openshift/library-go/pkg/proc/reaper.go

```diff
@@ -3,13 +3,87 @@
 package proc
 
 import (
+	"bufio"
+	"io/ioutil"
 	"os"
 	"os/signal"
+	"path/filepath"
+	"strconv"
+	"strings"
 	"syscall"
+	"time"
 
 	"k8s.io/klog"
 )
 
+// parseProcForZombies parses the current procfs mounted at /proc
+// to find proccess in the zombie state.
+func parseProcForZombies() ([]int, error) {
```

Comment:
Does the direct use of procfs couple this to Linux?

Reply:
We don't support containers on non linux right now - router wouldn't work on windows